tests/test_utils.py

```diff
@@ -488,12 +488,12 @@ def test_gather_dimensions_exceptions():
 
     # Attempt to reduce a non-existent dimension
     with pytest.raises(ValueError) as excinfo:
-        assert gd(fcst_dims_conflict, obs_dims, reduce_dims="nonexistent") == []
+        assert gd(fcst_dims_conflict, obs_dims, reduce_dims="nonexistent") == []  # pylint: disable=C1803
```

Comment:
What about something like
```python
output=gd(fcst_dims_conflict, obs_dims, reduce_dims="nonexistent") 
assert len(output) == 0
```
Also for a few lines down

Reply:
I've gone with the suggestion of pylint which is just 'assert not gd(...)'. The downside is that there are many things that evaluate to False which are not empty containters (e.g. not None --> True). A more specific test would probably be to assert both length and type. However the main purpose of this test is actually to check that the exception is raised properly, so I think we can be comfortable with a less specific test.